### ISSUE TITLE
Require Directory Structure

### ISSUE THREAD

**Issue by michaeltlombardi** [stlpsug/stlpsug#1]
The directory structure for this main repository should be simple, sane, and easy to follow.

**Comment by michaeltlombardi** [comment]
I've worked up a potential [directory structure](https://gist.github.com/michaeltlombardi/35e2141857f129301b3c):

```
.
| -- communication.md
| -- contributions.md
| -- current-events.md
| -- documents
|    | -- sponsorship.md
|    | -- steering.md
|    | -- ?
| -- events
|    | -- meetups
|    |    | -- 2016-January
|    |    |    | -- agenda.md
|    |    |    | -- presentation-vscode
|    |    |    |    | -- vscode-lab-setup.md
|    |    |    |    | -- vscode-presentation-notes.md
|    |    |    |    | -- vscode-snippets.md
|    |    |    | -- takeaways.md
|    |    | -- 2016-February
|    |    |    | -- agenda.md
|    |    |    | -- presentation-git-intro
|    |    |    |    | -- <files from presentation>
|    |    |    | -- presentation-psvisio
|    |    |    |    | -- <files from presentation>
|    |    |    | -- takeaways.md
|    |    | --2016-March
|    | -- automation-management-summit
|    |    | -- ams2016.md
|    | -- calendars
|    |    | -- 2016.md
| -- readme.md
| -- speakers
|    | -- bio-template.md
|    | -- johndoe.md
|    | -- janedoe.md
` -- topics.md
```

Thoughts?

**Edit:** Apparently, I can't embed a gist in a comment. Odd.

**Comment by michaeltlombardi** [comment]
Updated the directory structure to include a page for topics per @MikeShepard 's suggestion in #12.

**Comment by michaeltlombardi** [comment]
@doesitscript, does this seem to be a better layout for the event folder structure? I've adjusted it based on your work in #15.

```
| -- events
|    | -- meetups
|    |    | -- 2016
|    |    |    | -- january
|    |    |    |    | -- agenda.md
|    |    |    |    | -- presentation-vscode
|    |    |    |    |    | -- vscode-lab-setup.md
|    |    |    |    |    | -- vscode-presentation-notes.md
|    |    |    |    |    | -- vscode-snippets.md
|    |    |    |    | -- takeaways.md
|    |    |    | -- february
|    |    |    |    | -- agenda.md
|    |    |    |    | -- presentation-git-intro-josh-castillo
|    |    |    |    |    | -- <files from presentation>
|    |    |    |    | -- presentation-psvisio-mike-shepard
|    |    |    |    |    | -- <files from presentation>
|    |    |    | -- takeaways.md
|    |    |    | -- march
```

This way the meetups are broken out by year and then month but the presentations are kept logically connected to the same meetup they were at - so we can keep track of the agenda and takeaway documents as well.

**Comment by michaeltlombardi** [comment]
Closed by #15
